IntegrationTests/Braintree-API-Integration-Specs/BTAPIClient_IntegrationTests.m

```diff
@@ -1,6 +1,5 @@
-#import <BraintreeCore/BraintreeCore.h>
 #import <XCTest/XCTest.h>
-@import BraintreeCoreSwift;
+@import BraintreeCore;
```

Comment:
I remember this pain in the butt formatting we had to do for importing Swift into Obj-C when we first converted PPDataCollector over. We can talk about this pairing later, but IIRC using the `@import` won't work for all package managers.

https://github.com/braintree/braintree_ios/blob/master/DEVELOPMENT.md#importing-a-swift-module-into-obj-c

Reply:
I believe the `@import` should be okay for integration tests as the tests are passing currently. But I can update to `#import <BraintreeCore/BraintreeCore-Swift.h>` if that's our preferred way to import for these tests.